api/app/Models/WorkExperience.php

```diff
@@ -21,10 +22,25 @@
  * @property string $details
  * @property \Illuminate\Support\Carbon $created_at
  * @property \Illuminate\Support\Carbon $updated_at
+ * @property string $employment_category
+ * @property string $ext_size_of_organization
+ * @property string $ext_role_seniority
+ * @property string $gov_employment_type
+ * @property string $gov_position_type
+ * @property string $gov_contractor_role_seniority
+ * @property string $gov_contractor_type
+ * @property ?\Illuminate\Support\Carbon $gov_contract_start_date
+ * @property ?\Illuminate\Support\Carbon $gov_contract_end_date
```

Comment:
Sorry, this is my bad.  We don't need extra date fields for this.  We'll store the dates in the regular start and end dates of the experience.

Reply:
Nice.